libs/hdf-converters/src/gosec-mapper.ts

```diff
@@ -22,11 +22,29 @@ function nistTag(input: Record<string, unknown>): string[] {
   return CWE_NIST_MAPPING.nistFilter(cwe, DEFAULT_NIST_TAG);
 }
 
+// Check `nosec` and `suppressions` fields which denote whether the gosec rule violation should be suppressed/skipped
+function formatStatus(input: Record<string, unknown>): string {
+  return `${_.get(input, 'nosec')}` === 'false' &&
+    `${_.get(input, 'suppressions')}` === 'null'
+    ? ExecJSON.ControlResultStatus.Failed
+    : ExecJSON.ControlResultStatus.Skipped;
+}
+
+// If a gosec rule violation is suppressed, forward the given justification
+function formatSkipMessage(input: Record<string, unknown>): string {
+  return `${_.get(input, 'suppressions')}` !== 'null'
+    ? `${_.get(input, 'suppressions[0].justification')}`
```

Comment:
is it only an array of size 1?  should we just combine all the suppressions texts into a single string?

Reply:
Addressed.